docs/docs/processing-payments-with-square.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Processing Payments with Square
+---
+
+Square is a payment service that emphasizes quick, secure payments as well as user-friendly and affordable point of sale (POS) system. You may have already seen their tiny credit card readers, which are great for mobile businesses like those that sell via food trucks or craft fairs. This guide explains how to begin using Square with your Gatsby site.
+
+## Prerequisites
+
+You'll need to [set up a developer account](https://squareup.com/signup?v=developers) to get started. Create a new application from your developer dashboard. The name of your application cannot include the word "square". Once that's done, you should see your new application on your dashboard. Each application has an application ID and access token associated with it.
+
+## Choosing the right product
+
+Square can handle most of your payment-related needs including accepting payments, managing product catalogs, and managing payroll. Begin by determining which of these products you want to incorporate into your business. This guide focuses on [accepting payments on your website](https://developer.squareup.com/docs/online-payment-options#square-payments-in-your-own-website) but you may wish to use their in-person or in-app options as well.
+
+You've got two options for integrating Square payments: redirecting to a hosted checkout page with the Checkout API as well as using payments on your Gatsby site with the `SqPaymentForm` library and Payments API.
```

Comment:
```suggestion
You've got two options for integrating Square payments: redirecting to a hosted checkout page with the Checkout API or using payments on your Gatsby site with the `SqPaymentForm` library and Payments API.
```

Reply:
This change reads to me like you have a choice between Checkout + the `SqPaymentForm` library and the Payments API (which isn't so) versus having a choice between the Checkout API and the `SqPaymentForm` library + the Payments API.

Reply:
Hmm, I'm still a bit confused. Is it a choice between all three? Or are the SqPaymentForm library and the Payments API meant to be used together?
```suggestion
You've got two options for integrating Square payments: redirecting to a hosted checkout page with the Checkout API as well as using payments on your Gatsby site with the `SqPaymentForm` library and Payments API.
```

Reply:
SqPaymentForm and Payments API are meant to be use together. So maybe...
```suggestion
You've got two options for integrating Square payments: redirecting to a hosted checkout page with the Checkout API _or_ taking payments from your Gatsby site using the `SqPaymentForm` library with the Payments API.
```

Reply:
Perfect!